mozci/task.py

```diff
@@ -273,6 +273,22 @@ def configuration(self):
             get_test_variant(self.variant, self.label),
         )
 
+    @property
+    def is_potential_regression(self):
+        # Check if the job is either tier 1 or 2
+        if self.tier not in (1, 2):
+            return False
+
+        # Check if job result is busted or exception
+        if self.result not in ("busted", "exception"):
+            return False
+
+        # Check if extra.treeherder.jobKind is build
+        if self.job_kind != "build":
+            return False
+
+        return True
```

Comment:
We should add this logic within https://github.com/mozilla/mozci/blob/4c9faf8fa5ccdb2933125182d5d89b2dde2d9f34/mozci/push.py#L1128 (which is implementing classification for tests).
We could add another function before that called `classify_builds`, and call it within `classify_regressions`.

This way we can re-use the later logic for retriggers and backfills (which we might need to update).